### PR TITLE
Mentioned about the default kv engine per tenant

### PR DESCRIPTION
**Pull request by cuttingedge1109**

### CODE REVIEW

**Comment by stakater-user** [comment]
@cuttingedge1109 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-88-4c3c5ff3`

File: content/sre/secrets/vault.md [hunk 1]
```diff
@@ -35,7 +35,8 @@ There are 2 kinds of secrets in the vault.
   Users only have read permission.
   The path is `managed-addons/*`.
 * Tenant specific secrets.
-  Users can create/delete/update/read secrets on the `TENANT_NAME/*` path.
+  A `KV` v2 secret engine is enabled on `TENANT_NAME/kv` path in default. Even though users delete that path, it is created automatically so please don't remove that.
+  Users can enable/disable secret engines on `TENANT_NAME/*` paths and create/delete/update/read secrets in them.
```
Comment (rasheedamir) [inline content/sre/secrets/vault.md:40]: also add how developers can create secrets; i mean few screen shots

**Comment by stakater-user** [comment]
@cuttingedge1109 Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-88-82c53cab`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-openshift-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.47`

**Pull request merged**